mPower2/mPower2/StudyBurstScheduleManager.swift

```diff
@@ -437,9 +448,33 @@ class StudyBurstScheduleManager : TaskGroupScheduleManager {
         }
         
         let path = self.instantiateTaskViewModel(for: task)
+        
+        
+        if (steps.contains(where: { $0.identifier == CRFDemographicsKeys.birthYear.rawValue })) {
+            previousCrfResults()?.forEach({
+                path.taskViewModel.append(previousResult: $0)
+            })
+        }
```

Comment:
so... this works but... It's probably a bit easier to read if you do this instead (or something like it).
```
func setupDemographics(on task: RSDTask) {
    guard let crfTask = task as? CRFTaskObject,
            let yearAnswerStr = SBAProfileDataSourceObject.shared.profileTableItem(at: ProfileTableViewController.birthYearIndexPath)?.detail,
            let yearInt = Int(yearAnswerStr),
            let sexAnswerStr = SBAProfileDataSourceObject.shared.profileTableItem(at: ProfileTableViewController.sexIndexPath)?.detail  
    else {
        return nil
    }
    crfTask.gender = CRFGender(rawValue: sexAnswerStr)
    crfTask.birthYear = yearInt
}


```

Reply:
Agreed, reads better this way.